Yoast/wordpress-seo
# WIP Move sitemaps tests

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* N/A

## To do:
- Fix last 2 tests
- Add tests for build_sitemap and bad_sitemap

## Test instructions

This PR can be tested by following these steps:

* Run the tests and see them succeed

## UI changes

## Quality assurance


Fixes #
